src/map/entities/battleentity.h

```diff
@@ -234,13 +234,32 @@ enum SLOTTYPE
 // для возможности различить эти орудия при экипировке и избавиться от ошибки
 // использования пуль с арбалетом и арбалетных стрел с огнестрельным оружием (только персонажи)
 
+enum ATTACKTYPE
+{
+    ATTACK_NONE = 0,
+    ATTACK_PHYSICAL = 1,
+    ATTACK_MAGICAL = 2,
+    ATTACK_RANGED = 3,
+    ATTACK_SPECIAL = 4,
+    ATTACK_BREATH = 5,
+};
+
 enum DAMAGETYPE
 {
     DAMAGE_NONE = 0,
     DAMAGE_PIERCING = 1,
     DAMAGE_SLASHING = 2,
     DAMAGE_IMPACT = 3,
-    DAMAGE_HTH = 4
+    DAMAGE_HTH = 4,
+    DAMAGE_ELEMENTAL = 5,
+    DAMAGE_FIRE = 6,
+    DAMAGE_EARTH = 7,
+    DAMAGE_WATER = 8,
+    DAMAGE_WIND = 9,
+    DAMAGE_ICE = 10,
+    DAMAGE_LIGHTNING = 11,
+    DAMAGE_LIGHT = 12,
+    DAMAGE_DARK = 13,
```

Comment:
yet _another_ list of elements to keep track of where people can get the value of the element wrong because of other lists of elements with diff values..

I don't have a change request or suggestion here. just complaining to myself about the inevitable having to explain it to ppl later. we already have multiple and ppl mix them up.

Reply:
Yeah, I agree this is not preferable. Unfortunately I think the previous `DAMAGETYPE` values are in the database. I suppose something we could do is offset all of the previous ones so they start at 9 after the elements. That would allow us to use the existing element values as a `DAMAGETYPE`.

If this were to happen I'm not sure if changing the database or just using an offset when reading the database would be best.

Reply:
it should be fine the way it is - nobody should ever be interacting with this table with the raw numbers, always either this enum or the table on the lua side, so it shouldn't matter what the values actually are.  As long as we don't start using this enum in the database.

Reply:
>  As long as we don't start using this enum in the database.

at one time in the past we were. guess who nuked that.